x/metadata/spec/01_concepts.md

```diff
@@ -95,6 +95,16 @@ If the value owner address is changing as well as one or more other fields, thes
 * When a value owner address is a non-marker address, and is being changed, that existing address must be one of the signers.
 * When a value owner address is empty, and is being changed, standard scope signer requirements are also applied even if that's the only change to the scope.
 
+### Smart Contract Requirements
+
+The following are requirements related to smart contract usage of the `x/metadata` module:
+
+* A party with a smart contract address MUST have the `PROVENANCE` role.
+* A party with the `PROVENANCE` role MUST have the address of a smart contract.
+* When a smart contract signs a message, it MUST not have any non-smart-contract signers before it and SHOULD include the invoker address(es) after.
```

Comment:
Might be easier to understand if reworded...

When a smart contract signs a message, **it MUST have only smart-contract signers** before it and SHOULD include the invoker address(es) after.

Reply:
the only reason to not include the invoker is if the smart contract is the sole owner/manager of the scope/records.

Reply:
Much better. Updated.